colossalai/shardformer/shard/shard_config.py

```diff
@@ -57,3 +58,9 @@ def _turn_on_all_optimization(self):
         self.enable_fused_normalization = True
         self.enable_flash_attention = True
         self.enable_jit_fused = True
+
+    def _infer(self):
+        """
+        Set default params for inference.
+        """
+        self.pipeline_stage_manager = None
```

Comment:
What's the effect of this function?

Reply:
This is for setting some default configs for inference, and it's to be implemented in the future.

Reply:
Why do you set pipeline stage manager to `None`? If you don't pass pipeline stage manager to shard config, it's naturally `None`.

Reply:
I mean to explicitly remind that we don't us pipeline parallelism in inference, however , till now our infer infra do not offer other choices to use, like different kernels , may be we left it empty for now ? or just keep this here and replace it some time

Reply:
You can make an assertation instead of set to None. Setting it to none may lead to unknown error if there is pipeline stage manager.

Reply:
ok, I changed this